### PR TITLE
Added modBan Event

### PR DESCRIPTION
**Pull request by cosmetify**
more events are on their way ;)

### CODE REVIEW

File: docs/API.md [hunk 1]
```diff
@@ -2010,6 +2011,31 @@ mp.on('modAddDj', (data) => {
...
+Fired when a moderator bans a user.
+
+**Parameters**
+
+ - `moderator` - The staff [User](#class-user) who banned the user.
+ - `username` - The name of the user that was banned.
+ - `duration` - The duration of time the ban lasts for
+
+```js
+mp.on('modAddDj', (data) => {
```
Comment (cosmetify) [inline docs/API.md]: I need to improve my cp skillz :/

I'll wait to change this, since I'll probably have to change something else due to travis failing.

**Comment by goto-bus-stop** [comment]
Yeh, travis failure is just a semicolon again ;) Looking good otherwise, thanks!

**Comment by cosmetify** [comment]
Oh wow. Why do you do it like this ._.

(the 2 space indent is killing me)

**Comment by goto-bus-stop** [comment]
Good stuff! Thanks :D

**Pull request merged**